### ISSUE TITLE
[Bug]: [Accessibility][ChoiceGroup]: In scan mode, the narrator announces the ChoiceGroup option text twice.

### ISSUE THREAD

**Issue by sbatikeri** [microsoft/fluentui#24764]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.22000
    CPU: (8) x64 Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
    Memory: 13.03 GB / 31.83 GB
  Browsers:
    Edge: Spartan (44.22000.120.0), Chromium (105.0.1343.33)
    Internet Explorer: 11.0.22000.120
```

### Reproduction

https://codepen.io/sbatikeri/pen/abGmZZp
Video: https://user-images.githubusercontent.com/62160090/189744491-5a244bc8-8684-4289-8706-0cb348ec3980.mp4

## Actual Behavior
In scan mode, the narrator takes focus on 'option' and reads the option text. When we press down arrow again, the option text is focused and the narrator announces the option text twice.

## Expected Behavior
Option text should be announced only once

### Requested priority

Normal

### Products/sites affected

http://support.xbox.com/en-US/xbox-live-status

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by gouttierre** [comment]
This issue has been queued for upcoming investigations. To set expectations the developers will review this issue once capacity allows and all updates will be posted on the issue accordingly.

**Comment by sbatikeri** [comment]
Please update the bug.

**Comment by sbatikeri** [comment]
Please update

**Comment by miroslavstastny** [comment]
Each radio renders the following DOM structure:
```html
div
  div
    input
    label
      span
```

Which results in the following accessibility tree:
<img width="188" alt="image" src="https://user-images.githubusercontent.com/9615899/195632774-e2bc383e-1289-4fe8-9000-87341f54c745.png">
In this case the Narrator in scan mode announces both the `radio` and `Label text`.

If I remove the `span` element from the `label` and place its content directly into the label the accessibility tree changes as follows:
<img width="195" alt="image" src="https://user-images.githubusercontent.com/9615899/195633256-d565bdf7-7390-41e1-8905-d16454553e50.png">

And Narrator reads the text only once. That is the behavior of v9.

**Comment by sbatikeri** [comment]
When will be avilable V9?

**Comment by JustSlone** [comment]
Agree with Miro, as I recall, this behavior is either a Narrator bug or By Design. One of the purposes of v9 is to fix longstanding issues like these. @smhigley may be able to confirm. 

ChoiceGroup is replace with RadioGroup in v9 and is available here: https://react.fluentui.dev/?path=/docs/components-radiogroup--default

Check the migration guide for how to and implications of mixing v8 and v9 components. Good news is that you can pickup a few v9 controls without updating your whole app, though v9 uses a different styling and theming system, so you'll see a bundle size increase mixing the two.

**Comment by smhigley** [comment]
Hi all!

This isn't really a bug -- Narrator is reading both the radio button and the text because there is both a radio button and label text in the accessibility tree.

Edge is inserting an additional text node into UIA's control view when the inner label text is wrapped in a separate node, and may also do so for other reasons, e.g. certain label styling, or nesting an interactive item in the label text. I wouldn't really call this a bug, since there are valid reasons a browser may need to expose a text node in the accessibility tree. If a link were nested inside the label, for instance, Narrator would need to be able to separately access the label text.

Some screen readers do additional heuristic detection to reduce verbosity, e.g. by not reaching both the label text and form control in cases like these. NVDA does not reach both the radio and the label, for instance (though NVDA also uses a different accessibility API from Narrator). Narrator does not do this, but also isn't wrong not to do so -- it relies much more directly on Edge's UIA mapping.

TL:DR; the expected result outlined above is not always actually expected -- it isn't a bug for screen readers to separately reach a control and its label text. The behavior differs by screen reader and browser, and also by the content of the label.

**Comment by sbatikeri** [comment]
Thanks @smhigley @JustSlone
